### ISSUE TITLE
json.go file is inconsistent

### ISSUE THREAD

**Issue by shaojunda** [nervosnetwork/ckb-sdk-go#201]
The file  `types/json.go` in release v2.1.1 does not match the code of file A in the repository.

in release

```
func (r *TransactionWithStatus) MarshalJSON() ([]byte, error) {
	jsonObj := &jsonTransactionWithStatus{
		Transaction: r.Transaction,
		Cycles:      hexutil.Uint64(r.Cycles),
		TxStatus:    r.TxStatus,
	}
	return json.Marshal(jsonObj)
}
```

in repo:
https://github.com/nervosnetwork/ckb-sdk-go/blob/2cbfd9eb7cd8828bcb847a8b3c7f0b074d049edd/types/json.go#L765-L775

**Comment by code-monad** [comment]
Will fix in next release after #202  merged

**Comment by code-monad** [comment]
Fixed in v2.2.0
